Lib/idlelib/idle_test/test_textview.py

```diff
@@ -8,12 +8,12 @@
 Coverage: 94%.
```

Comment:
After I adapt my coverage.bat to work with the git repository, I will get  new value.